### PR TITLE
Jarmode execution full pipeline. (one configured for cf-6282, others will be added one by one)

### PR DESCRIPTION
**Pull request by tahiat**

### CODE REVIEW

**Comment by tahiat** [comment]
existing compilation check script could have been modified support jar mode, but instead copied and made separate script

File: check_jar_mode_compilation.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
```
Comment (kelloggm) [inline check_jar_mode_compilation.sh:1]: Are there any differences between this script and the approximate mode checking script other than the name of the output .json file? If not, I think it's much better to only have one script and make that a parameter.

**Review by kelloggm — CHANGES_REQUESTED** [review]

File: main.py [hunk 2]
```diff
@@ -13,10 +13,13 @@
 import tarfile
 import glob
 import stat
+import argparse
 
 issue_folder_dir = 'ISSUES'
 specimin_input = 'input'
 specimin_output = 'output'
+specimin_jar_output = 'jar_output'
+specimin_jar_output = 'jar_output'
```
Comment (kelloggm) [inline main.py:22]: Duplicate line

**Comment by tahiat** [comment]
Professor, I added dependency pulling scripts (not in this pR) for few projects in respective forked repos. 
Projects where I have not added the jar pull scripts:
daikon-1291 | calcite - 3619, 3850 | beam- 6388

**Review by kelloggm — APPROVED** [review]
I don't need to review again after the one thing is fixed.

**Pull request merged**